Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -1442,10 +1442,29 @@ void FrameLoader::loadFrameRequest(FrameLoadRequest&& request, Event* event, Ref
         }
     };
 
-    if (request.resourceRequest().httpMethod() == "POST"_s)
-        loadPostRequest(WTFMove(request), referrer, loadType, event, WTFMove(formState), WTFMove(completionHandler));
-    else
-        loadURL(WTFMove(request), referrer, loadType, event, WTFMove(formState), WTFMove(privateClickMeasurement), WTFMove(completionHandler));
+    auto finishLoadFrameRequest = [this, referrer, event, loadType, frame, completionHandler] (FrameLoadRequest&& request, RefPtr<FormState>&& formState, std::optional<PrivateClickMeasurement>&& privateClickMeasurement) mutable {
```

Comment:
Don't capture this without a something managing the lifetime of this.  Here, I think you should capture a Ref<LocalFrame> and call LocalFrame::loader

Reply:
Done, however it turns out due to recent changes we need stuff like https://bugs.webkit.org/show_bug.cgi?id=274170 in place to make this show up green. Patrick is working on that, but for now we can hold off action on this PR and also no need to run the tests on the bot yet.

Reply:
#28576